runtime/gear/src/lib.rs

```diff
@@ -223,13 +167,17 @@ impl frame_system::Config for Runtime {
     type SS58Prefix = SS58Prefix;
     /// The set code logic, just the default since we're not a parachain.
     type OnSetCode = ();
-    type MaxConsumers = frame_support::traits::ConstU32<16>;
+    type MaxConsumers = ConstU32<16>;
+}
+
+parameter_types! {
+    pub const MaxAuthorities: u32 = 32;
 }
```

Comment:
```suggestion
const MAX_CONSUMERS: u32 = 16;
{    
    // .. // 
    
    type MaxConsumers = ConstU32<MAX_CONSUMERS>;
}

const MAX_AUTHORITIES: u32 = 32;

// .. //
impl pallet_aura::Config for Runtime {
     type AuthorityId = AuraId;
     type DisabledValidators = ();
     type MaxAuthorities = ConstU32<MAX_AUTHORITIES>;
 }
```

Reply:
Not sure this is better. What problem would it solve?
I would rather put `MaxConsumers` into `parameter_types!` macro just for the sake of uniformity. I guess it's more about personal preference than actual optimization, is it not?